test/integration/select_readview_test.lua

```diff
@@ -187,14 +188,15 @@ pgroup.test_close_gc_on_router = function(g)
         collectgarbage("collect")
         collectgarbage("collect")
     ]])
+    fiber.sleep(1)
     t.assert_equals(err, nil)
     local res = {}
-    helpers.call_on_storages(g.cluster, function(server, replicaset, res)
+    helpers.call_on_storages(g.cluster, function(server)
         local instance_res = server.net_box:eval([[
         return box.read_view.list()]])
-        res[replicaset.alias] = instance_res
+        table.insert(res, instance_res)
      end, res)
-    t.assert_equals(res, {["s-1"] = {}, ["s-2"] = {}})
+     t.assert_equals(res, {{}, {}, {}, {}})
```

Comment:
This is wrong indentation (and at the previous line too(.

Reply:
changed